tests/DataSourceTest.php

```diff
@@ -482,7 +481,7 @@ public function testExtensionsCalls()
     }
 
     /**
-     * @return \PHPUnit_Framework_MockObject_MockObject
+     * @return DriverInterface|MockObject
```

Comment:
Just `MockObject`, otherwise Scrutinizer will report this as an issue. Please correct other instances of dual class docblocks for mocks as well.